src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -1,5 +1,23 @@
 import { Typography, Grid, Card, Box } from '@mui/material'
 import { WidgetBody, WidgetContainer } from '@/components/dashboard/styled'
+import SafeWidget from '@/components/dashboard/SafeWidget/SafeWidget'
+
+export type WidgetType = {
+  appId: string
+  appUrl: string
+  widgetType?: 'iframe'
+}
+
+export const appWidgets: WidgetType[] = [
+  {
+    appId: '1',
+    appUrl: 'https://swap.cow.fi/',
+  },
+  {
+    appId: '2',
+    appUrl: 'https://revoke.cash',
+  },
+]
```

Comment:
I've added a tag for you on staging: `dashboard-mini-app`
You can use that to query these instead of the ids.

Reply:
Oh! Nice! Thanks